types/keytar/index.d.ts

```diff
@@ -10,9 +10,9 @@
  * @param service The string service name.
  * @param account The string account name.
  *
- * @returns the string password or null on failures.
+ * @returns Promise, which on success yields the password string or error string on failures.
```

Comment:
I think this message can be tidied up - the error message should be included in the promise when it fails, and isn't specialized to what keytar is doing here.